### PR TITLE
Add exercise "Say"

### PR DESCRIPTION
**Pull request by zporter**
This on behalf of [Triangle |> Elixir](https://www.meetup.com/Triangle-Elixir/) ❤️

### CODE REVIEW

**Comment by parkerl** [comment]
Hey @zporter thanks for the PR! Are you working on the `example.exs` for these tests? You will also need to add this exercise to `config.json` in order for Travis to go green.

**Comment by zporter** [comment]
@parkerl -- absolutely!  We were working during our meetup last night and had to end before we could finish this. I'm planning on working in the example tonight, having my friend review, then finishing this PR up.  Thanks for checking in!

**Comment by zporter** [comment]
@parkerl -- I think we're good to go here. Please let me know if there's anything I can do to help.  Thanks!

File: exercises/say/say_test.exs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  Code.load_file("say.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule SayTest do
+  use ExUnit.Case
+
+  test "says zero in english" do
```
Comment (devonestes) [inline exercises/say/say_test.exs]: Typically we actually include the `@tag` line before the first test and comment it out. It's a nice indicator to the user as to how they're supposed to move through the test. Think you could add this?

Comment (zporter) [inline exercises/say/say_test.exs]: Made the change and updated the branch. Thanks!

File: exercises/say/say_test.exs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule SayTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "says zero in english" do
```
Comment (parkerl) [inline exercises/say/say_test.exs]: @zporter can you update the test descriptions to match https://github.com/exercism/x-common/blob/master/exercises/say/canonical-data.json

File: exercises/say/say_test.exs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    end
+  end
+
+  @tag :pending
+  test "numbers above 999_999_999_999 are out of range" do
+    assert_raise ArgumentError, fn ->
+      Say.in_english(1_000_000_000_000)
+    end
+  end
+
```
Comment (parkerl) [inline exercises/say/say_test.exs]: remove extra line please

File: exercises/say/say_test.exs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  @tag :pending
+  test "says a big number in english" do
+    assert Say.in_english(987_654_321_123) == "nine hundred eighty-seven billion six hundred fifty-four million three hundred twenty-one thousand one hundred twenty-three"
+  end
+
+  @tag :pending
+  test "numbers below zero are out of range" do
+    assert_raise ArgumentError, fn ->
```
Comment (parkerl) [inline exercises/say/say_test.exs]: @zporter we have been embracing the use of `{:ok, result}` or `{:error, "message"}` idiom in most of our newer exercises. Can you please update this exercise to that style?

File: exercises/say/say_test.exs (hunk 2 above)
Comment (zporter) [inline exercises/say/say_test.exs]: @parkerl, like, in this instance, `test "zero" do`?

File: exercises/say/say_test.exs (hunk 4 above)
Comment (zporter) [inline exercises/say/say_test.exs]: Sure, @parkerl.  Just for a little clarification, for the successful cases, we'd have `Say.in_english(1_000_000_000) == {:ok, "one billion"}` and failed cases, we'd have `Say.in_english(-1) == {:error, "number is out of range"}`?

File: exercises/say/say_test.exs (hunk 3 above)
Comment (zporter) [inline exercises/say/say_test.exs]: Done

File: exercises/say/say_test.exs (hunk 4 above)
Comment (devonestes) [inline exercises/say/say_test.exs]: Yep - that's the right idea!

File: exercises/say/say_test.exs (hunk 2 above)
Comment (parkerl) [inline exercises/say/say_test.exs]: Yes thank you!

**Comment by zporter** [comment]
Thanks for the feedback, @parkerl! I've made the requested changes. Please let me know if there's anything else I can do to help!

**Pull request merged**